content/en/docs/zero-code/java/spring-boot/starter/getting-started.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Getting Started
+description: Getting Started of the OpenTelemetry starter
+weight: 20
+# prettier-ignore
+cSpell:ignore: autoconfigurations springboot
+---
+
+## OpenTelemetry Spring Boot starter
+
+### Compatibility
+
+The OpenTelemetry Spring Boot starter works with Spring Boot 2.0 and 3.0, and
+Spring Boot native image applications. The
+[opentelemetry-java-examples/spring-native](https://github.com/open-telemetry/opentelemetry-java-examples/tree/main/spring-native)
+repository contains an example of a Spring Boot Native image application
+instrumented using the OpenTelemetry Spring Boot starter.
+
+### Dependency management
+
+A Bill of Material
+([BOM](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#bill-of-materials-bom-poms))
+ensures that versions of dependencies (including transitive ones) are aligned.
+
+Importing the `opentelemetry-bom` and `opentelemetry-instrumentation-bom-alpha`
+BOMs when using the OpenTelemetry starter is important to ensure version
+alignment across all OpenTelemetry dependencies.
+
+The following example shows how to import both BOMs using Maven:
+
+```xml
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-bom</artifactId>
+            <version>{{% param vers.otel %}}</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.opentelemetry.instrumentation</groupId>
+            <artifactId>opentelemetry-instrumentation-bom-alpha</artifactId>
+            <version>{{% param vers.instrumentation %}}-alpha</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+    </dependencies>
+</dependencyManagement>
+```
+
+With Gradle and Spring Boot, you have
+[two ways](https://docs.spring.io/spring-boot/docs/current/gradle-plugin/reference/htmlsingle/)
```

Comment:
the link is rather confusing

Reply:
It does not seem related to the reorganization of the Spring Boot content. You could perhaps elaborate this point in a Github issue?